config/settings.yml

```diff
@@ -0,0 +1,10 @@
+links:
+  rails_tutorial:
+    home_page: http://www.railstutorial.org/
+    help_page: http://www.railstutorial.org/help
+    help_book: http://www.railstutorial.org/book
+    contact: https://www.railstutorial.org/contact    
+    news: https://news.learnenough.com
+    author: https://www.michaelhartl.com
+    screencast: https://www.learnenough.com/courses
+    ruby_on_rails: https://rubyonrails.org/
```

Comment:
please read it
https://docs.google.com/document/d/104Csp4-vamVos5DEbi372nLBEfmqhj7h1ENYO8ebjCo/edit
and add endline same as No 12